genestack_client/version.py

```diff
@@ -1,2 +1,2 @@
-__version__ = '0.5.2a1'
+__version__ = '0.5.2a2'
```

Comment:
also, there is an application, that checks the python-client version. We should bump it there too